### DISCUSSION TITLE
Can we use WithLeopardGF16(true) when shards are less than 256?

### DISCUSSION THREAD

**Discussion by ptbxzrt** [klauspost/reedsolomon#236] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/89020404/215327248-ed1b0e6a-2fe8-4589-809e-9b17e4a655f0.png)
There is a comment like above. But in the test codes, it test encode with gf16 when shards are less than 256. So I wonder whether I can use WithLeopardGF16(true) when shards are less than 256.
![image](https://user-images.githubusercontent.com/89020404/215327327-e233d766-26d4-4d86-b5b7-9a8ad57884cf.png)

**Comment by klauspost** [comment, marked as answer]
I means it is incompatible with [Leopard-RS](https://github.com/catid/leopard) implementation.

You should use `WithLeopardGF(true)` which will use [Leopard GF8](https://github.com/klauspost/reedsolomon#leopard-gf8) which is both faster and compatible.
